internal/cli/kraft/cloud/volume/remove/remove.go

```diff
@@ -36,16 +39,25 @@ func Remove(ctx context.Context, opts *RemoveOptions, args ...string) error {
 
 func NewCmd() *cobra.Command {
 	cmd, err := cmdfactory.New(&RemoveOptions{}, cobra.Command{
-		Short:   "Permanently delete a persistent volume",
-		Use:     "remove UUID [UUID [...]]",
-		Args:    cobra.MinimumNArgs(1),
-		Aliases: []string{"rm"},
+		Short:   "Remove volumes",
```

Comment:
Can we please keep the phrase `Permanently delete a persistent volume` and instead:
```
Permanently delete persistent volume(s)
```
As this phraseology will be consistent to OpenAPI.